core/test/fixtures/fraggle-rock/reports/sample-flow-result.json

```diff
@@ -1797,45 +1797,8 @@
             "displayValue": "",
             "details": {
               "type": "opportunity",
-              "headings": [
```

Comment:
This delta here shows the new logic is WAI, as this image is directly requested by the main document.

<img width="1260" alt="image" src="https://user-images.githubusercontent.com/4071474/191582051-6431290f-59d0-4cd9-92c2-d5a0f1ba128f.png">

https://www.mikescerealshack.co/corrections